### ISSUE TITLE
Can not login

### ISSUE THREAD

**Issue by grandmasteriVan** [MicrosoftDocs/feedback#1044]
**Describe the bug**
On page https://docs.microsoft.com/en-us/learn/ I click 'Sing in' link. Than I login with my microsoft account and redirect on page https://techprofile.microsoft.com/en-us/register?replyUrl=https://docs.microsoft.com/en-us/learn/& Whan I enter any name I get 'You are already registered' message.
I already have techprofile of https://techprofile.microsoft.com/en-us/grandmasteriVan

**Screenshots**
![2019-01-14 16 17 26](https://user-images.githubusercontent.com/14936499/51118811-e9f66980-1819-11e9-9082-0f99e13b0c56.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.14.2
 - Browser: safari
 - Version 12.0.2 (14606.3.4)

**Comment by Gamecock** [comment]
I am seeing the similar issue:
https://techprofile.microsoft.com/en-us/mikefinch87

When I click on signin it takes me to register:
https://techprofile.microsoft.com/en-US/register?replyUrl=https://techprofile.microsoft.com/en-us/mikefinch87&
I then get:
![image](https://user-images.githubusercontent.com/5841226/51951942-7a30d300-2404-11e9-896c-8d70b33e9a39.png)


I can make this a separate issue if you want.

Desktop (please complete the following information):

OS: MacOs 10.12.6
Browser Chrome
Version 71.0.3578.98 (Official Build) (64-bit)

**Comment by UintraContact** [comment]
have the same issue! any new here? really annoying issue, cannot started with learning and certification here :(

**Comment by tomkirish** [comment]
I am currently experiencing this issue as well. I cannot sign into my account regardless of browser and/or machine. I changed some of my settings in my Microsoft account and constantly get this same behavior.

**Comment by saldana** [comment]
@grandmasteriVan Are you still experiencing the issue? We released a fix few days ago that should have this issue fixed. 

If you are still experimenting the issue, can you let me know if you can log in from an InPrivate/Incognito window?

**Comment by UintraContact** [comment]
issues fixed seems to me, thanks

**Comment by saldana** [comment]
Thank you for your response, @UintraContact. I will consider the issue fixed then as other users confirmed the fix also resolved their problem.

**Comment by sandyvistan** [comment]
I'm experiencing same issue started today.

**Comment by Gamecock** [comment]
@saldana This issue is back for me today.  I tried logging out of all microsoft accounts and back in, and closing browsers.  I was able to log in yesterday.

**Comment by saldana** [comment]
@jialge and @Sampy can you guys look into this?

**Comment by Sampy** [comment]
@sandyvistan @Gamecock Have either of you made any of the following changes to your Microsoft Account recently?

- Changed the default email address mapped to the account on account.live.com
- Added or removed an email address or phone number from the account on account.live.com
- Used the "Sign in with GitHub" feature on the Microsoft Account sign in screen
- Any other changes to your Microsoft Account you can think of

From our logs, we see you arriving with the same sign in email address (which we do not save or use as an identifier) as before but a different ID from the Microsoft Account system which means we treat you as a different user. The cases above are the most likely causes of this sort of issue.

**Comment by Gamecock** [comment]
I did not change this account. I changed work accounts to a new account.
Mike_Finch@jabil.com.
I might have added a phone number.

Mike

**Comment by Gamecock** [comment]
On the main "Microsoft Learning" pagehttps://www.microsoft.com/en-us/learning/default.aspx I am logged in as me.
![image](https://user-images.githubusercontent.com/5841226/54822861-52c7eb00-4c7c-11e9-99b3-3862bdf87849.png)
When I go here: https://docs.microsoft.com/en-us/learn/  I get logged out. 
![image](https://user-images.githubusercontent.com/5841226/54822901-75f29a80-4c7c-11e9-8232-6443470b1969.png)
Clicking on the login link takes me to the create profile page.
I checked my profile, and it only has 1 email and no phone numbers.

**Comment by Sampy** [comment]
Short answer: this is expected behavior. If you are failing to create a profile after you click Sign In on Learn, that's an issue we should keep investigating.

(Apologies if you know all of this already. This information may be useful to others so I wanted to share it)

Long answer: Each of these sites is using your personal Microsoft Account (MSA) as their identity provider but they each need to write cookies to their own domains to know that you are logged in. By signing in with your MSA, you have cookies for login.live.com (and the site you signed in to) but not every site that could use MSA. For sites that allow anonymous access like MS Learn, if you arrive after having signed in to another MSA using site you will still see a sign in button. You won't see a login page at all (since your cookies from login.live.com will ensure they know who you are) but MS Learn still needs to know by getting a POST from your browser. You can see this in action on other sites by logging in at account.live.com and then going to xbox.com. Since xbox.com allows anonymous access, you still have to click a sign in link

Now the portal does not allow anonymous access. Because of that, it will send any anonymous request over to the identity provider. Since you have cookies for that domain, you'll wont see a sign in screen at all and you'll get what appears to be a single sign on experience. This is actually the only way to achieve this sort of experience (send all users to the identity provider) which is usually not a great user flow for the anonymous user.

We show you a registration page because we need information specific to our site to build your profile page. That information is required to create the page and start tracking your learning so when you arrive for the first time we need to ask for it.

---

Now if you arrive at that profile page, fill it out, and click the Register button and it fails then that *is* a problem and we can work to investigate that. This isn't the only report we have on this so we are actively working the problem.

**Comment by Gamecock** [comment]
@Sampy You are correct, the issue is that I have too many accounts/profiles.  I will need to delete one.
